lib/script-generator.js

```diff
@@ -195,12 +196,11 @@ ScriptGenerator.prototype.exclude = function (rule, index) {
     '# }'
   ].join('\n');
 
-  var excludes = rule.files.map(function (name) {
-    return '\\( ! -name \'' + name + '\' \\)';
-  }).join(' ');
+  var excludes = rule.files.filter(isString).map(function (name) {
+    return 'grep -v \'^./' + name.replace(/'/g, '') + '$\'';
+  }).join(' | ');
```

Comment:
this is kind of hard to follow. mind commenting it ?
what I see is it creates a string from a file array
the file array is first filtered to ensure it is a string
then with that list you generate a grep -v lines which excludes those files (removing `'` from the name)